setup.py

```diff
@@ -53,7 +53,8 @@
                      'torch>=1.5.0,<1.6.0'],
         'cliner': ['marisa-trie==0.7.4', 'scipy==1.5.0',
                    'scikit-learn==0.23.1', 'repoze.lru==0.7',
-                   'tensorflow-gpu==1.12.0', 'python-crfsuite==0.9.7']
+                   'tensorflow-gpu==1.12.0', 'python-crfsuite==0.9.7'],
+        'transformers': ['transformers==3.3.1']
```

Comment:
The key here is to indicate a type of function. Here you can probably use `data_aug` or `back_translate` as the key.